api/api/templates/admin/base_site.html

```diff
@@ -0,0 +1,22 @@
+{% extends "admin/base_site.html" %}
+
+{% block messages %}
+    {% if ENVIRONMENT != 'production' %}
+    <script>
+        function calculateDaysLeft() {
+            let dateToday = new Date(Date.UTC(new Date().getUTCFullYear(), new Date().getUTCMonth(), new Date().getUTCDate()));
+            let lastDayOfMonth = new Date(Date.UTC(dateToday.getUTCFullYear(), dateToday.getUTCMonth() + 1, 0)).getUTCDate();
+            let daysUntilEndOfMonth = lastDayOfMonth - dateToday.getUTCDate();
+            document.getElementById("staging-db-refresh-days").innerText = daysUntilEndOfMonth;
+        }
+        document.addEventListener("DOMContentLoaded", calculateDaysLeft);
+    </script>
+    <ul class="messagelist">{% for message in messages %}
+        <li{% if message.tags %} class="{{ message.tags }}"{% endif %}>{{ message|capfirst }}</li>
+    {% endfor %}
+        <li class="warning">Next staging database restore will occur in <span id="staging-db-refresh-days"></span> days.
+            <a href="https://airflow.openverse.engineering/dags/staging_database_restore/grid">View the DAG for information on how to skip this process</a>
```

Comment:
Adding the full stop.
```suggestion
            <a href="https://airflow.openverse.engineering/dags/staging_database_restore/grid">
                View the DAG for information on how to skip this process.
            </a>
```